### DISCUSSION TITLE
Add Phone/Call support

### DISCUSSION THREAD

**Discussion by xbyte83** [twentyhq/twenty#9081] [Ideas]
- I'm missing one very crucial feature, and that is phone support. Integrate with Twilio or AirCall; there's plenty.
- Register, actions on calls, length of conversations etc..

**Comment by meteoDaniel** [comment]
We Use Placetel, they support the webex API and have their own one. 

We would really appreciate such feature 🙌🏽 

BTW.: Thanks for this awesome project !
